tests/hypervisor/test_esx.py

```diff
@@ -1432,7 +1433,67 @@ def test_post_large_json_to_rhsm(self, register_data, ssh_host):
         else:
             logger.warning("Failed to post json to satellite")
             logger.warning(output)
-
+    
+    @pytest.mark.tier2
```

Comment:
should be `@pytest.mark.tier3`
and remove the `@pytest.mark.notStage`, we should test this case both satellite and stage